.github/DESIGN.md

```diff
@@ -15,6 +26,32 @@ The user understands the basic mechanics of a CLI:
 - CTRL + C quits any running task
 - how to navigate and manipulate the filesystem via the command line: cd and mkdir
 
+## Components of a well designed command
+When creating a new command or subcommand there are a few things to keep in mind.
+
+### Reduce user input
+When trying to create parity with an existing GUI based feature, consider the larger task at hand and build your command to accomplish that goal. 
+
+For example when creating a `dev-store` the data model requires up to 8 fields to be filled in. When creating a CLI command to create a `dev-store`, don't ask the user for all 8 fields - auto-fill all 8 fields with smart generated data. If a field requires an address dont generate an animal name, try to get as close as possible to the format the data expects.
```

Comment:
I'm not sure we should use `dev-store` as an example here as we haven't implemented it yet

Reply:
Agree I removed it, and went with a simpler example